sdk/storage/azure_storage_blob/src/clients/blob_client.rs

```diff
@@ -2,22 +2,22 @@
 // Licensed under the MIT License.
 
 use crate::{
-    models::BlobProperties, pipeline::StorageHeadersPolicy, BlobBlobClientGetPropertiesOptions,
-    BlobClientOptions, GeneratedBlobClient,
+    generated::clients::blob_client::BlobClient as GeneratedBlobClient, models::BlobProperties,
```

Comment:
Ideally this will be a non-issue:
`generated::clients::blob_client::BlobClient as GeneratedBlobClient` and we won't have to reimport it as a different name once the collision is resolved with generated names.